entries.json

```diff
@@ -568,4 +568,38 @@
 		"no generics",
 		"no exceptions"
 	]
+},{
+	"URL": "https://bluxte.net/musings/2018/04/10/go-good-bad-ugly/",
+	"Author": "Sylvain Wallez",
+	"Year": 2018,
+	"Complaints": [
+		"no generics",
+		"sort.Interface approach is clumsy",
+		"designers did not consult the literature",
+		"Fisher-Price assembly language",
```

Comment:
LOL! I like the term even if I didn't imply it was simple/bad: just very surprising :-)